Cargo.lock

```diff
@@ -2057,14 +2057,20 @@ version = "0.52.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "733954023c0b39602439e60a65126fd31b003196d3a1e8e4531b055165a79b31"
 
+[[package]]
+name = "wasmparser"
+version = "0.55.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "af931e2e1960c53f4a28b063fec4cacd036f35acbec8ff3a4739125b17382a87"
+
 [[package]]
 name = "wasmprinter"
 version = "0.2.4"
```

Comment:
I just published 0.2.5, mind updating that here to try to only use one wasmparser version?

Reply:
Sure thing :+1: